tests/test_strat.py

```diff
@@ -75,38 +75,35 @@ def _batch_jacobian(output, input_):
     return torch.stack(jacs, dim=0)
 
 
-def _gdg_jvp_brute_force(sde, t, y, a):
+def _dg_ga_jvp_brute_force(sde, t, y, a):
     # Only returns the value for the first input-output pair.
```

Comment:
Is this referring to the re-rewrite potential of having tuples? If so it can be removed.

Reply:
No, this is simply a rename, i.e. `gdg_jvp` -> `dg_ga_jvp`. I think the latter is clearer, since it matches the math better.

The rationale of including this function is for testing purposes, i.e. we have more confidence in the two variants of jvp I implemented, and it would come in handy in the future if we ever decide to modify any aspect of those functions.

Reply:
I mean the comment, not the name!

Reply:
Ah you right, good point!